app/models/deploy.rb

```diff
@@ -135,6 +135,27 @@ def duration
     updated_at - start_time
   end
 
+  # time from when PRs were created until get deployed
+  def total_lead_time
+    changeset.pull_requests_lead_time + duration
```

Comment:
this math does not make sense to me ... total-lead-time would be `prs.sum { deploy.end - pr.start }`